### DISCUSSION TITLE
How to install vc_redist in nanoserver ?

### DISCUSSION THREAD

**Discussion by atulthummar** [dotnet/dotnet-docker#3178] [Q&A]
How to install vc_redist in nanoserver?

#
#FROM mcr.microsoft.com/dotnet/runtime:5.0-nanoserver-1809 AS base
#WORKDIR /app

**Comment by atulthummar** [comment]
finally i have identified that it is not possible in nanocore image so used server core image to install VC++ dependency

**Reply by Dipsonmick** [reply]
how do i install nanocore ?
